### PR TITLE
Drop deprecated code and revise log handling

### PR DESCRIPTION
**Pull request by bl-young**
- old code for `flowbysector.py` is dropped
- deprecated functions are removed (primarily in `validation.py`, `dataclean.py`, `fbs_allocation.py`)
- data_source_script cleaning functions which now use more general approaches have been removed (e.g., `BEA.py`
- `temp_data_source_functions.py` have been moved to appropriate scripts
- logging functions are moved to `flowsa_log.py` (from `settings.py`) and duplicate code removed
   - log files are enabled for FBA and FBS (to be refined in future)

### CODE REVIEW

File: flowsa/dataclean.py [hunk 1]
```diff
@@ -231,91 +230,3 @@ def standardize_units(df):
...
-    df = df.assign(Spread=None)
-    # reset min, as current values are misleading
-    log.info('Reset Min to None')
-    df = df.assign(Min=None)
-    # reset min, as current values are misleading
-    log.info('Reset Max to None')
-    df = df.assign(Max=None)
-    # reset DR, as current values are misleading
-    log.info('Reset DataReliability to None')
-    df = df.assign(DataReliability=None)
```
Comment (catherinebirney) [inline flowsa/dataclean.py:307]: @bl-young Do we still want to reset the DQ scores to None?

Comment (bl-young) [reply flowsa/dataclean.py:307]: We can certainly add that back in to `flowby.py` - it never made it in there. I can do that.

Comment (bl-young) [reply flowsa/dataclean.py:307]: ok completed with 82af6df

**Review by catherinebirney — APPROVED** [review]
Thanks looks good, this can be pulled in

**Pull request merged**